### DISCUSSION TITLE
Matrix code random white/bright leading character like movie.

### DISCUSSION THREAD

**Discussion by ThaPchild** [rocksdanister/lively#2308] [General]
I love the matrix rain wallpaper.  It's amazing on my super ultra wide.  

Is there any way we can get random white leading characters like on the movie?  Would love to see it a little more accurate.  It took me a second to realize what was missing when installed but then it hit me, there's no white first character or haze around it.  Possible with HTML5?

You can find the scene on the you tubez if you look for "Neo and Cypher talk"  I'll attach a screen as well.

![NeoMatrix](https://github.com/rocksdanister/lively/assets/123424800/34560c14-e991-44d2-98a6-482ce2fc85c7)

**Comment by ThaPchild** [comment]
Speed Randomization for individual lines/strings would also be amazing!

Love the screen saver, thanks!

**Comment by Gunz52** [comment]
Even though this is a bit of an old post, I made a modification* to the matrix script to have trails at different speeds and with the first character of the code in white.

*Not true, I didn’t make it, Claude did. XD

[script.js](https://github.com/user-attachments/files/22730630/script.js)
